pnpm-workspace.yaml

```diff
@@ -0,0 +1,7 @@
+packages:
+  # include packages in subfolders (change as required)
+  - 'packages/*'
+  # if required, exclude directories
+  - '!**/test/**'
+  - '!**/build/**'
+  - '!**/compile/**'
```

Comment:
Why it is this needed?

Reply:
I excluded folders not to be used to resolve workspace dependencies. I'm not sure it's terribly useful. There were examples on the site and I have given some of them

Reply:
In my opinion, their example is applicable in cases where you have sub-test suites (thus, with a package.json, but used for running tests).

Reply:
removed